test/controllers/InvestmentRegulatedSchemeControllerSpec.scala

```diff
@@ -59,6 +59,12 @@ class InvestmentRegulatedSchemeControllerSpec extends ControllerWithQuestionPage
       InvestmentRegulatedSchemeId,
       true
     )
+
+    "set the IsAboutBenefitsAndInsuranceCompleteId to false to change to in progress status" in {
```

Comment:
Think should put a must clause round this else it doesn't read right as it's inside a when clause